src/TMap.cpp

```diff
@@ -2599,10 +2600,10 @@ void TMap::slot_setDownloadProgress(qint64 got, qint64 tot)
     if (!mpProgressDialog->maximum()) {
         // First call, range has not been set;
         mpProgressDialog->setRange(0, mExpectedFileSize);
-    } else if (tot != -1 && mpProgressDialog->maximum() != static_cast<int>(tot)) {
-        // tot will stuck at -1 when we do not know how big the download is
+    } else if (total != -1 && mpProgressDialog->maximum() != static_cast<int>(total)) {
+        // total will stuck at -1 when we do not know how big the download is
```

Comment:
Oops, I made a spelling mistake when I wrote the original text:
```suggestion
        // total will stick at -1 when we do not know how big the download is
```